src/main/java/com/cloudbees/jenkins/GitHubPushTrigger.java

```diff
@@ -234,7 +255,12 @@ public String getUrlName() {
         }
 
         public String getLog() throws IOException {
-            return Util.loadFile(getLogFile());
+            File logFile = getLogFile();
+            if (null == logFile) {
+                return "";
```

Comment:
Better to throw an `IOException`.

Reply:
Good catch, I think I updated some of the changes to throw, and clearly not others.  I'll update this position.